pkg/proto/internals/v1/service_invocation_additional.go

```diff
@@ -0,0 +1,23 @@
+/*
+Copyright 2023 The Dapr Authors
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package internals
+
+const daprSeparator = "||"
+
+// This file contains additional, hand-written methods added to the generated objects.
+
+// GetActorKey returns the key for the actor.
+func (x *Actor) GetActorKey() string {
```

Comment:
This function and the file name seem to be mismatched. Would something with `actor` in it make more sense? I want to avoid misleading future developers when they go to look at this file for `service_invocation_additional` and find a `GetActorKey` func

Reply:
The `Actor` struct is generated by the protobuf message `Actor` defined in [`internal/v1/service_invocation.proto`](https://github.com/dapr/dapr/blob/6966/dapr/proto/internals/v1/service_invocation.proto). This means that the struct itself is written to the auto-generated file `pkg/proto/internals/v1/service_invocation.pb.go`

The file is named consistently with the name of the file where the struct is. We also use the `<proto>_additional.go` pattern in [at least one more instance](https://github.com/dapr/dapr/blob/6966/pkg/proto/runtime/v1/dapr_additional.go), so it's consistent with that.

Reply:
Thanks for explaining. I see the [dapr_additional.go](https://github.com/dapr/dapr/tree/master/pkg/proto/runtime/v1) 👍🏻